README.md

```diff
@@ -43,12 +43,19 @@ are supported:
 
 ```javascript
 WebVRConfig = {
+
+  // Flag to disabled the UI in VR Mode
```

Comment:
I would be specific and mention the back button and the settings gear and the eye line. also should end with a period to be consistent with punctuation of sentences elsewhere.